src/client/indexform/index.js

```diff
@@ -32,8 +32,12 @@ function IndexForm() {
     label: 'Fall',
     id: '19F',
   });
-  // Const [showResults, setShowResults] = useState(true);
-  // const [courses, setCourses] = useState([]);
+  const [selectedSubjArea, setSelectedSubjArea] = useState({
+    id: 'ENGCOMP',
```

Comment:
@rlorenzo Yes that's correct. I have not hardcoded the value but assigning the default value... in case subjArea is missing. 
If you don't want me to handle it that way, I can still delete it.

Reply:
@tatikamsetti If no subject area is passed, it should just be a blank value.